extension/lib/display.js

```diff
@@ -80,13 +79,14 @@ class Display {
     */
     updateTimer(readableDomIds, containerId) { // Call everytime you get to new paragraph
         // TODO: Should also update timer if a user is using the autoread mode. 
-        let total_char = 0; // TODO: REFACTOR TOTAL_CHAR CALCULATION
-        for (var section in readableDomIds.slice(containerId)) {
-            let elem = document.getElementById(readableDomIds[section]);
-            let text_len = elem.innerText.length;
-            total_char = total_char + text_len;
+        let total_words = 0;
+        let remainingContainers = readableDomIds.slice(containerId); // Need to store as own new list, so for loop indexes through this, not old list
+        for (var section in remainingContainers) {    // Calc total words
+            let text = $("#" + remainingContainers[section]).text();
+            let wordRegex = /\b[^\d\W]+\b/gi; // Checks for words that don't include numbers or non-letters
```

Comment:
hrm should we not keep this logic consistent with what we have for calculating tracker life?
https://github.com/benfrancis314/reading_project/blob/master/extension/content_script.js#L113

Currently, it's # chars

Example user experience that i thought we're going for -- but lmk what you were aiming for
*Sees estimated time: 10m"
" oh boy thats too long, lemme speed it up"
*increase speed*
" Sees estimated time: 5m"
"ok cool if i autoplay now, in 5m i should be done reading"